chromium/background.js

```diff
@@ -712,4 +716,4 @@ Object.assign(exports, {
   urlBlacklist,
 });
 
-})(typeof exports == 'undefined' ? window.background = {} : exports);
+})(typeof exports == 'undefined' ? require.scopes.background = {} : exports);
```

Comment:
Use `module.exports`. `exports` doesn't always work.

Reply:
`typeof module.exports` would throw a `ReferenceError` because `module` is not defined.